lib/utils/yieldSourceFormatters.jsx

```diff
@@ -0,0 +1,80 @@
+import { getYieldSourceName } from '@pooltogether/utilities'
+
+import { PRIZE_POOL_TYPE } from 'lib/constants'
+
+import CompSvg from 'assets/images/comp.svg'
+
+export const DEFAULT_STAKE_SOURCE_NAME = '--'
+export const DEFAULT_CUSTOM_YIELD_SOURCE_NAME = 'Custom Yield Source'
+export const COMPOUND_FINANCE_SOURCE_NAME = 'Compound Finance'
```

Comment:
Aren't the cream pools reusing the compound logic inthe smart contracts? Do they end up labelled properly?

Reply:
Compound logic but different contracts / contract names. ie. Rari calls them f tokens (instead of c). We don't want the names showing up as "Compound Finance" though, so this works as a good way to differentiate. The data for which tokens / yield sources are deployed where is in `@pooltogether/utilties`